opal/mca/event/libevent2022/libevent/evdns.c

```diff
@@ -3108,11 +3106,11 @@ evdns_search_ndots_set(const int ndots) {
 
 static void
 search_set_from_hostname(struct evdns_base *base) {
-	char hostname[HOST_NAME_MAX + 1], *domainname;
+	char hostname[MAXHOSTNAMELEN], *domainname;
```

Comment:
FWIW, we import libevent from an external package.  We typically modify as little code in this libevent core code as possible (because then we'll just need to remember to make the same updates the next time we upgrade the internal copy of libevent).

Specifically: it's probably ok / harmless to make these changes as part of this PR, but we should probably expect them to drift back the next time we upgrade libevent.